MSAL/src/public/MSALPublicClientApplication.h

```diff
@@ -34,50 +34,156 @@
 
 @interface MSALPublicClientApplication : NSObject
 
+/*! Whether or not to validate the authority used with the application, default YES */
 @property BOOL validateAuthority;
+
+/*! The authority used with the application object */
 @property (readonly) NSURL *authority;
+
+/*! The client ID of the application */
 @property (readonly) NSString *clientId;
+
+/*! The redirect URI of the application */
```

Comment:
That's covered by the quick start documentation and the error that comes back if the developer hasn't registered it.